lib/src/metta/runner/stdlib/string.rs

```diff
@@ -25,7 +25,7 @@ impl CustomExecute for PrintlnOp {
     fn execute(&self, args: &[Atom]) -> Result<Vec<Atom>, ExecError> {
         let arg_error = || ExecError::from("println! expects single atom as an argument");
         let atom = args.get(0).ok_or_else(arg_error)?;
-        println!("{}", atom_to_string(atom));
+        println!("{}", atom_into_string(atom.clone()));
```

Comment:
Cloning here is ineffective.
1. Remove old `atom_to_string` implementation
2. Rename `atom_into_string` to `atom_to_string`
3. Change `atom_to_string` signature to `atom_to_string(atom: &Atom)` to eliminate cloning

Reply:
Done